prisma/seed.ts

```diff
@@ -0,0 +1,252 @@
+          { name: "Spaghetti", quantity: 200, unit: Unit.GRAM },
+          { name: "Pancetta or bacon", quantity: 100, unit: Unit.GRAM },
+          { name: "Eggs", quantity: 3, unit: Unit.PIECE },
+          { name: "Parmesan cheese", quantity: 50, unit: Unit.GRAM },
+          { name: "Black pepper", quantity: 1, unit: Unit.PINCH },
+        ],
+      },
+      tag: {
+        connect: [{ name: "Quick" }, { name: "Lunch" }],
+      },
+    },
+  });
+
+  const pancakes = await prisma.recipe.create({
+    data: {
+      name: "Pancakes",
+      slug: "pancakes",
+      description:
+        "Fluffy, golden-brown pancakes perfect for breakfast. Serve with syrup, fruits, or whipped cream!",
+      steps: [
+        "In a bowl, whisk together flour, sugar, baking powder, and salt.",
+        "In a separate bowl, whisk together milk, eggs, and melted butter.",
+        "Make a well in the dry ingredients, then pour in the wet ingredients.",
+        "Stir until just combined. Be careful not to overmix, as this will result in dense pancakes.",
+        "Heat a non-stick pan over medium heat and lightly grease with butter or oil.",
+        "Pour about 1/4 cup of batter onto the pan for each pancake.",
+        "Cook for 2-3 minutes on one side, until bubbles appear on the surface.",
+        "Flip and cook for another 1-2 minutes until golden brown.",
+        "Serve with your favorite toppings, like maple syrup, fresh berries, or whipped cream.",
+      ],
+      images: [
+        "https://assets.tmecosys.com/image/upload/t_web767x639/img/recipe/ras/Assets/9767DACF-96E6-417B-9BE9-DA57EAC9462D/Derivates/2f26d615-1da2-43dd-a39e-6a54742c9299.jpg",
+      ],
+      visibility: "PUBLIC",
+      authorId: admin.id,
+      categoryId: 1,
+      ingredients: {
+        create: [
+          { name: "Flour", quantity: 200, unit: Unit.GRAM },
+          { name: "Sugar", quantity: 2, unit: Unit.TABLESPOON },
+          { name: "Baking powder", quantity: 2, unit: Unit.TEASPOON },
+          { name: "Salt", quantity: 0.25, unit: Unit.TEASPOON },
+          { name: "Milk", quantity: 250, unit: Unit.MILLILITER },
+          { name: "Eggs", quantity: 2, unit: Unit.PIECE },
+          { name: "Butter (melted)", quantity: 30, unit: Unit.GRAM },
+        ],
+      },
+      tag: {
+        connect: [{ name: "Breakfast" }, { name: "Quick" }],
+      },
+    },
+  });
+
+  const adminFavorites = await prisma.recipeList.create({
+    data: {
+      name: "Favorites",
+      isSystem: true,
+      userId: admin.id,
+      recipes: {
+        connect: [{ id: koreanChicken.id }, { id: spaghettiCarbonara.id }],
+      },
+    },
+  });
+
+  const userFavorites = await prisma.recipeList.create({
+    data: {
+      name: "Favorites",
+      isSystem: true,
+      userId: user.id,
+      recipes: {
+        connect: [{ id: pancakes.id }],
+      },
+    },
+  });
+
+  await prisma.shoppingList.create({
+    data: {
+      ingredients: ["Eggs", "Milk", "Flour", "Sugar"],
```

Comment:
I think this will need to be changed because I changed ingredients in shopping bag from strings into Ingredients